tests/GenerateScriptTests/GenerateScriptTests.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GenerateScriptTests;
+
+public class GenerateScriptTests
+{
+    public enum PackageType
+    {
+        Reference,
+        Text
+    }
+
+    public static IEnumerable<object[]> Data => new List<object[]>
+    {
+        new object[] { "System.Xml.ReaderWriter", "4.0.11", PackageType.Reference },
+        new object[] { "Microsoft.Extensions.Logging.Abstractions", "7.0.1", PackageType.Reference },
+        new object[] { "Microsoft.CodeAnalysis.CSharp", "3.11.0", PackageType.Reference },
+        new object[] { "System.Security.Cryptography.Pkcs", "7.0.2", PackageType.Reference },
```

Comment:
We should be validating text only packages as well.  All the test infrastructure was defined but no test data for them were added.
```suggestion
        new object[] { "System.Security.Cryptography.Pkcs", "7.0.2", PackageType.Reference },
        new object[] { "Microsoft.Build.NoTargets", "3.7.0", PackageType.Text },
```